### PR TITLE
Style changes to python and HTML

### PR DESCRIPTION
**Pull request by pamelafox**
This PR makes the following improvements, mostly style related:

* Ran black on the Python code for formatting
* Ran isort on the Python code for import sorting
* Ran pylint on the Python code to identify problematic code
* Ran the HTML through W3C validator and Lighthouse to identify HTML validity and a11y issues

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -1,3 +0,0 @@
-
```
Comment (pamelafox) [inline pytest.ini:1]: The pytest autodiscovery mechanisms already looks for either test_ or _test, so this config isn't needed.
https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#test-discovery

File: templates/home.html [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
-            $(window).load(function () {
-                $("body").removeClass("preload");
-            });
-        </script>
-        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
-            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
-            crossorigin="anonymous"></script>
+    </div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (pamelafox) [inline templates/home.html:28]: This is just indentation change after adding the missing div close tag above.

File: templates/home.html [hunk 3]
```diff
@@ -1,38 +1,40 @@
...
+    <link rel='stylesheet' href='../static/stylesheets/style.css'>
 </head>
 
 <body class="preload">
     <div class="centered">
 
-        <button type="button" class="btn btn-lg btn-toggle" data-toggle="button" aria-pressed="false" autocomplete="off"
-            onclick="togglePaw()" id="main-button">
-            <div class="handle"></div>
+        <button type="button" class="btn btn-lg btn-toggle" data-toggle="button" aria-pressed="false"
```
Comment (pamelafox) [inline templates/home.html:16]: button tags require some sort of text that can be read by a screen-reader

File: templates/home.html [hunk 4]
```diff
@@ -1,38 +1,40 @@
...
+            onclick="togglePaw()" id="main-button" title="A button for cats">
+            <span class="handle"></span>
```
Comment (pamelafox) [inline templates/home.html:18]: divs cant be inside buttons, but spans can, and your CSS seems compatible with spans too.

File: templates/home.html [hunk 5]
```diff
@@ -1,38 +1,40 @@
...
         </button>
         <div class="catpaw-container">
-            <img class="catpaw-image" alt="myImage" src="../static/images/catpaw.png" />
+            <img class="catpaw-image" alt="The paw of a cat" src="../static/images/catpaw.png">
```
Comment (pamelafox) [inline templates/home.html:21]: The alt text wasn't very descriptive

File: templates/home.html [hunk 6]
```diff
@@ -1,38 +1,40 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
     <title>A Very Useful Website</title>
-    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
```
Comment (pamelafox) [inline templates/home.html:6]: W3C discourages ending slashes on void tags

**Comment by pamelafox** [comment]
@luabud Please review, thank you!

**Comment by luabud** [comment]
Amazing, thank you so much! 🥳

**Pull request merged**